FourthTask/sorting/construct/construct.go

```diff
@@ -0,0 +1,62 @@
+//it works, but i have one problem there
```

Comment:
Still don't understand what the reason for this file

Reply:
it's just a construct with user interface. i thought that it's cool idea to work with goroutines in this file.